0miner

```diff
@@ -158,7 +158,12 @@ then
     HCD="${NVOC}/miners/dstm/latest/zm_miner --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --port ${!xport} --pass $MINER_PWD --time $DSTM_OPTS ${!xext}"
   elif [[ $EQUIHASH_MINER == EWBF ]]
   then
-    HCD="${NVOC}/miners/ewbf/$EWBF_VERSION/miner --eexit 3 --fee $EWBF_PERCENT --pec --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --pass $MINER_PWD --port ${!xport} $EWBF_OPTS ${!xext}"
+    if [[ EWBF_VERSION == 3_4 || EWBF_VERSION == 3_3 ]]
+    then
+      HCD="${NVOC}/miners/ewbf/$EWBF_VERSION/miner --eexit 3 --fee $EWBF_PERCENT --pec --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --pass $MINER_PWD --port ${!xport} $EWBF_OPTS ${!xext}"
+    else
+      HCD="${NVOC}/miners/ewbf/$EWBF_VERSION/miner --fee $EWBF_PERCENT --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --pass $MINER_PWD --port ${!xport} $EWBF_OPTS ${!xext}"
```

Comment:
There may be a problem here, The coint->algo binding in 1bash associates BTG to "EQUIHASH", but WTM is calling the new algo "EquihashBTG" https://whattomine.com/coins/214-btg-equihash
Should we consider each equihash variant as a different algo?

Reply:
If we separate algos, the problem with switching divverent variants is addressed also, we would have:
old equihash->old ewbf (like it is in current tree)
equihashbtg, and other variants -> new ewbf

Reply:
Actually WTM totally removed BTG from its coin calculation page, may add a new equihash algo for it... or not.
So lets wait and see how it goes.